### PR TITLE
perf: @talismn/balances-react performance fixes

### PR DESCRIPTION
**Pull request by alecdwm**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +27,67 @@
     "pretty-quick": "^3.1.3"
   },
   "resolutions": {
+    "@polkadot/api-augment": "9.10.5",
```
Comment (alecdwm) [inline package.json:30]: Need those pesky `resolutions` to get around `@substrate/txwrapper-core`'s lack of dependency range etiquette

Comment (alecdwm) [reply package.json:30]: Feel free to find another way if you can :)

File: apps/web/src/libs/talisman/index.tsx [hunk 2]
```diff
@@ -45,11 +47,16 @@ const Provider = ({ children }: PropsWithChildren) => (
   <Extension.Provider>
     <Chainmeta.Provider>
       <Account.Provider>
-        <Balance.Provider>
-          <Parachain.Provider>
-            <Crowdloan.Provider>{children}</Crowdloan.Provider>
-          </Parachain.Provider>
-        </Balance.Provider>
+        <BalancesProvider
+          balanceModules={balanceModules}
```
Comment (alecdwm) [inline apps/web/src/libs/talisman/index.tsx:51]: balance module params moved to here

**Review by tien — APPROVED** [review]
lgtm

**Pull request merged**